src/manifest.ts

```diff
@@ -1,26 +1,44 @@
+import { accessSync } from 'fs'
 import { readFile } from 'fs-extra'
 import * as path from 'path'
 import { memoize } from 'ramda'
 
 import { CommandError } from './errors'
 
 const readFileUtf = async (file: string): Promise<string> => {
-  try {
-    return await readFile(file, 'utf8')
-  } catch (e) {
-    throw new CommandError(`Manifest file doesn't exist or is not readable. Please add a manifest.json file in the root of the app folder.`)
+  return await readFile(file, 'utf8')
+}
+
+export const manifestFileName = 'manifest.json'
```

Comment:
⛏ `MANIFEST_FILE_NAME` perhaps

Reply:
I've kept in camel case because some other constants in this file are also defined with that casing, but I can change this one